media/201710-presentation-devcon3.md

```diff
@@ -216,10 +216,10 @@ These rules reach into the network state and load/store from account storage:
 ``` {.k .uiuck .rvk}
```

Comment:
This should just be:

`````
```k
`````

Reply:
Can you look for other occurances of uiuck and rvk and fix those ones too?

Reply:
Oh, actually, I just saw this was for the DevCon presentation. please undo those changes then, we sholud not modify the presentation as it was given